pkg/catalogv2/content/content_test.go

```diff
@@ -0,0 +1,204 @@
+package content
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/rancher/rancher/pkg/settings"
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/repo"
+)
+
+func TestFilterReleases(t *testing.T) {
+	tests := []struct {
+		testName               string
+		chartVersionAnnotation string
+		rancherVersion         string
+		expectedPass           bool
+	}{
+		{
+			"rancher version in range comparison with `>= <`style comparison",
+			">= 2.5.0-alpha3 <2.6.0",
+			"2.5.0-alpha2",
+			true,
```

Comment:
Shouldn't this be false ? `2.5.0-alpha3 - 2.6.0`. Either I'm mistaken or this is passing incorrectly.

Reply:
Good catch and you are correct. I had missed the `v` in the version, so it was not getting past the ```	if !settings.IsRelease() {
		return index
	}``` check so it was never actually evaluating the comparison. I have made changes to correct this.